### ISSUE TITLE
Increase compatibility with Apple/Google specifications

### ISSUE THREAD

**Issue by dspinellis** [eellak/epidose#16]
Currently we're implementing the Apple/Google Bluetooth beacon format. The specifications are more extensive. Find, prioritize, and implement areas where compatibility would improve the device's usefulness and adoption.

**Comment by stefanos1316** [comment]
Are these the [specifications](https://covid19-static.cdn-apple.com/applications/covid19/current/static/contact-tracing/pdf/ExposureNotification-BluetoothSpecificationv1.2.pdf) you refer to?

**Comment by dspinellis** [comment]
Indeed!  These are one of the few documents where the two company logos appear together.  You might want to open separate issues for each thing that needs to be implemented, or if they are simple, you can add a checkbox list in this issue.
